extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -84,6 +84,17 @@ const provider: extensionApi.Provider = {
   onDidUpdateDetectionChecks: vi.fn(),
 };
 
+// Use 'provider', but just replace status with 'ready'
+const providerWithReadyStatus = {
+  ...provider,
+  status: 'ready' as extensionApi.ProviderStatus,
+};
+
+const providerWithStoppedStatus = {
+  ...provider,
+  status: 'stopped' as extensionApi.ProviderStatus,
```

Comment:
Weird that we need to cast ? Isn't stopped a valid ProviderStatus ?

Reply:
I suppose this cast is to cast the entire object? Would be best to cast providerWithStoppedStatus itself